project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+                'activity_task_type': 'activity',
+                # add room_id
+                'notes': act.notes,
+                'child_ids': [
+                    (0,
+                     0,
+                     {
+                         'name': task.task_name,
+                         'activity_task_type': 'task',
+                         'task_responsible_id': task.task_resp_id.id,
+                         'task_category_id': task.task_category_id.id,
+                         'department_id': task.department_id.id,
+                         'notes': task.notes,
+                         # add resource_id
+                     }
+                     ) for task in tasks],
+            }
+            activity = self.env['project.task'].create(activity_vals)
+
+    @api.onchange('template_id')
+    def _onchange_template_id(self):
+        template = self.env['event.template'].browse(self.template_id.id)
+        self.name = template.name
+        self.event_resp_id = template.temp_resp_id
+        self.event_notes = template.notes
+
+
+class ActivityTemplateLineWiz(models.TransientModel):
+    _name = "activity.template.line.wiz"
+    _description = "Activity Template Line Wizard"
+
+    event_wiz_id = fields.Many2one(
+        'project.event.wizard',
+        string='Activity Template',
+    )
+    template_id = fields.Many2one(
+        'activity.template',
+        string='Activity Template',
+    )
+    name = fields.Char(
+        string='Activity Title',
+    )
+    activity_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    activity_category_id = fields.Many2one(
+        'activity.category',
+        string='Category',
+    )
+    room_id = fields.Many2one(
+        'resource.resource',
+        string='Room',
+    )
+    notes = fields.Text(string='Notes')
+
+
+class TaskTemplateLineWiz(models.TransientModel):
+    _name = "task.template.line.wiz"
+    _description = "Task Template Line Wizard"
+
+    event_wiz_id = fields.Many2one(
+        'project.event.wizard',
+        string='Event Wizard',
+    )
+    activity_wiz_id = fields.Many2one(
+        'activity.template.line.wiz',
+        string='Activity wizard',
+    )
+    template_id = fields.Many2one(
+        'task.template',
+        string='Task Template',
+    )
+    task_name = fields.Char(
+        string='Task Title',
+    )
+    task_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    task_category_id = fields.Many2one(
+        'task.category',
+        string='Category',
+    )
+    resource_id = fields.Many2one(
+        'resource.resource',
+        string='Resource',
+    )
+    department_id = fields.Many2one(
+        'hr.department',
+        string='Department',
+    )
+    duration = fields.Float(
+        string='Duration',
+    )
+    start_time = fields.Float(
+        string='Start Time',
+    )
+    notes = fields.Text(string='Notes')
```

Comment:
@rimbendhaou: [Suggestion] Apply the same format as other field defined in the model.